pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/io/params/RecommenderConstants.java

```diff
@@ -119,4 +127,5 @@
   public static final int FIRST = 0;
   public static final int SECOND = 1;
 
+  public static final int DEFAULT_NUM_ROWS_IN_GENERATED_SEGMENT = 50_000;
```

Comment:
I am guessing the reason to move this outside is because we use this in both RT provisioning rule and SegmentSizeRule  ?

Reply:
Actually, that is not true. Looks like this is being used only for RT provisioning rule param ? In that case, move it inside lines 86 - 92 ?

Reply:
Ignore. Discussed offline. Both rules use the same default and can be configured independently.